include/boost/multiprecision/mpfi.hpp

```diff
@@ -284,8 +284,8 @@ struct mpfi_float_imp
          return *this;
       }
 
-      BOOST_MP_ASSERT(!(boost::math::isinf)(a));
-      BOOST_MP_ASSERT(!(boost::math::isnan)(a));
+      BOOST_MP_ASSERT(!(boost::multiprecision::detail::isinf)(a));
```

Comment:
Argument is __float128, as before update BOOST_MP_FLOAT128_USING and make an unqualified call.